model/task_lifecycle_test.go

```diff
@@ -822,6 +824,8 @@ func TestUpdateVersionStatusForGithubChecks(t *testing.T) {
 
 func TestUpdateBuildAndVersionStatusForTaskAbort(t *testing.T) {
 	require.NoError(t, db.ClearCollections(task.Collection, task.OldCollection, build.Collection, VersionCollection, event.LegacyEventLogCollection))
+	require.NoError(t, db.CreateCollections(task.Collection, task.OldCollection))
```

Comment:
same comment about creating task collection for all the other create collections function you added